spopt/tests/test_spenc.py

```diff
@@ -22,38 +22,38 @@ def test_labels_(self):
         observed_labels = self.model.labels_
         known_labels = np.array(
             [
-                3,
-                1,
-                4,
-                1,
-                1,
-                3,
-                1,                
-                4,
-                1,
-                1,
-                0,
-                0,
+                4
```

Comment:
Do we know that this is the correct solution?

Reply:
I do not know whether it is correct. just want to make sure the solution is the same with one random seed.

Reply:
OK, then we'll definitely need a review from @sjsrey and @ljwolf on this.